koku/koku/celery.py

```diff
@@ -119,6 +119,11 @@ def task(self, *args, **kwargs):
     "args": [],
 }
 
+# task to clean up sources with `pending_delete=t`
+app.conf.beat_schedule["delete_source_beat"] = {
+    "task": "sources.tasks.delete_source_beat",
+    "schedule": crontab(hour="4"),
```

Comment:
Can we make this an environment variable?

Reply:
Discussed offline. Decided to keep this value hardcoded. Task will run once per day at 4 am.